docs/releases/1.9.txt

```diff
@@ -124,6 +124,12 @@ levels of graceful degradation.
 
 .. _YUI's A-grade: http://yuilibrary.com/yui/docs/tutorials/gbs/
 
+PostgreSQL backend renamed
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The PostgreSQL backend was renamed from ``django.db.backends.postgresql_psycopg2``
```

Comment:
"was renamed from" -> "is also available from ... The old name will continue to be available for backwards compatibility."